tests/MigrationTraitTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use Exception;
+use Illuminate\Support\Facades\Config;
+use RonasIT\Support\Tests\Support\Mock\Migration\Migration;
+use RonasIT\Support\Tests\Support\Traits\SqlMockTrait;
+
+class MigrationTraitTest extends TestCase
+{
+    use SqlMockTrait;
+
+    protected Migration $migration;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        self::$tables = null;
+
+        $this->migration = new Migration();
+    }
+
+    public function testChangeEnum()
+    {
+        $this->mockStatementDBFacade('ALTER TABLE some_table DROP CONSTRAINT some_table_enum_field_check');
+
+        $this->mockStatementDBFacade(
+            'ALTER TABLE some_table '.
+            'ADD CONSTRAINT some_table_enum_field_check '.
+            "CHECK (enum_field::text = ANY (".
+                "ARRAY['first_value'::character varying, 'second_value'::character varying]::text[]".
+            '))'
+        );
```

Comment:
may we do so method like this?

```suggestion
        $this->mockStatementDBFacade(
            'ALTER TABLE some_table ADD CONSTRAINT some_table_enum_field_check CHECK (enum_field::text = ANY ('.
                "ARRAY['first_value'::character varying, 'second_value'::character varying]::text[]".
            '))'
        );
```